src/components/Common/Statement.js

```diff
@@ -26,13 +26,23 @@ const Link = styled(StyledLink)`
   display: initial;
 `
 
+const StyledStrong = styled.span`
+  color: ${({ theme }) => theme.colors.text};
+  font-weight: 500;
+`
+
+const makeStrong = text => <StyledStrong>{text}</StyledStrong>
+
+const mapWithBold = arr =>
+  arr.split('__').map((el, i) => (i % 2 === 1 ? makeStrong(el) : el))
+
 const Statement = ({ richText, children, as = 'h2' }) => (
   <GreyBackground>
     <PaddedGrid>
       <Row>
         <Col width={[1, 1, 1, 10 / 12, 10 / 12, 9 / 12]}>
           <DisplayTitle as={as} textLight>
-            {children}
+            {children && mapWithBold(children)}
             {richText &&
```

Comment:
regarding this twisted logic in `Statement` component; It's a tricky one and trickier solution.

it turns out that encapsulating a text with `__` in Contentful gives a bold text in the preview; but not in our apps.
The logic assumes that a bold text will always have an impair index since a split occurs when we meet __;

In other words, it would not work if someone would right `__bold_text1__ __bold_text2__` in Contentful !
Hopefully no one would because to make everything bold we would have `__bold_text1 bold_text2__` that works.
`__bold_text1__, __bold_text2__` would also work with as `,` would not render as bold.


This is not an idea solution but it looks like the Learnerbly case study is the only place where we do such a thing at the moment.

Reply:
I would probably opt for something like [this](https://gist.github.com/omonk/a9b7e5f7d285183027d869f6b04affc1)

We have an issue with the Statement component here as we pass it two different types of data, richtext formatted data from contentful and markdown from `longText`, (I'm not sure why we have two different types as we only use the richtext to render anchors  as `<Link/>` which we can do with react-markdown but aaaaanyway).

It's a bit of big change to a Common component but from doing some checks I believe it might be the preferred solution. Reasons for this are: 
- We're utilising libraries and tools we already have in the project
- If we didn't have these tools already `mapWithBold` is a good solution to this issue but what you've started on is entering the world of syntax parsing, something thats notoriously time consuming and tricky to handle so I would let the react-markdown package handle all of that for us.

Downsides of this solution: 
- You have to pass headers and paragraphs as the same renderer incase different markdown types were passed to a field in contentful.